src/jit/compiler.h

```diff
@@ -7607,7 +7607,7 @@ public :
         /* hide/trivialize other areas */                                                       \
                                                                                                 \
         STRESS_MODE(REGS) STRESS_MODE(DBL_ALN) STRESS_MODE(LCL_FLDS) STRESS_MODE(UNROLL_LOOPS)  \
-        STRESS_MODE(MAKE_CSE) STRESS_MODE(INLINE) STRESS_MODE(CLONE_EXPR)                       \
+        STRESS_MODE(MAKE_CSE) STRESS_MODE(LEGACY_INLINE) STRESS_MODE(CLONE_EXPR)                \
```

Comment:
I see two references to preprocessor constant STRESS_INLINE.  Post renaming it as STRESS_LEGACY_INLINE don't we need to rename in importer.cpp as well?  Looks like you might have forgotten to include/push importer changes.